### DISCUSSION TITLE
Set max number of words per line?

### DISCUSSION THREAD

**Discussion by Kirk2ko** [SubtitleEdit/subtitleedit#7641] [General]
I'd like to set the max number of words per line. The number of characters per line should be a secondary priority. So I'm talking about some kind of a rule setting, for example:

Max WPL (words per line): 3
Max CPL: 14

"I do want that" = WPL 3, CPL 14 = good
"It would do it" = WPL 4, CPL 14 = not good

Is this possible?

**Comment by LearningJer** [comment]
I like this idea as well! Something similar to what Davinci Resolve Studio offers. Here is an example I found.
https://www.youtube.com/watch?v=iOOw7pQ9_3M
